Automattic/wp-calypso
# Staging Sites: dont show banners in activity log screen on sync

Related to https://github.com/Automattic/dotcom-forge/issues/4268

## Proposed Changes

This PR aims to fix a bug where after consecutive syncs, the success banner is shown in the activity log.
We want to hide this banner when a rewind is triggered as a result of a sync.

### High level changes
- Rename `restoreId` property, to `lastRestoreId`, and store that in the state.

## Testing Instructions
1. Apply  D125961-code
2. Create a WoA site with a staging site.
3. Initiate a sync.
4. Navigate to `Activity Log` page. Ensure that you don't see a banner ( Check the `sync-state` endpoint. You should not be able to see any banners in the whole syncing process (status becomes 'completed')
5. Navigate back to the `/hosting-configuration` page.
6. Initiate another sync.
7. Navigate to the activity log page.
8. Ensure that no banner is visible.



## Pre-merge Checklist
